benchmark/common/result.py

```diff
@@ -85,8 +102,34 @@ def f1(self) -> float:
     def f1(self, f1) -> None:
         self._f1 = f1
 
+    def _plus(self, a: Optional[float], b: Optional[float]) -> Optional[float]:
+        if a is None or b is None:
+            return None
+        return a + b
+
+    def _minus(self, a: Optional[float], b: Optional[float]) -> Optional[float]:
+        if a is None or b is None:
+            return None
+        return a - b
+
+    def _divide(self, a: Optional[float], b: Optional[float]) -> Optional[float]:
+        if a is None or b is None or a == 0 or b == 0:
```

Comment:
```suggestion
        if a is None or b is None or b == 0:
```

Reply:
Is it okay for only b is 0 case? I think you want to cover a is 0 case too..